datasets/_externals/brightquery.yml

```diff
@@ -0,0 +1,58 @@
+type: external
+title: BrightQuery Business Identity
```

Comment:
Maybe either "BrightQuery Business Identity API" or  "BrightQuery Business Directory" or something like that? Like this it doesn't sound so nice.